src/main/java/org/junitpioneer/jupiter/params/ByteRangeSource.java

```diff
@@ -36,6 +37,7 @@
 @Documented
 @ArgumentsSource(RangeSourceArgumentsProvider.class)
 @RangeClass(ByteRange.class)
+@Repeatable(ByteRangeSource.ByteRangeSources.class)
```

Comment:
As these annotations are now repeatable and can be used with Cartesian products, I think they need some more changes:

* explain their dual use in their Javadoc (including a link to Cartesian product docu)
* explain where they can be repeated and where they can't